new_platforms/nuget/build/native/openenclave.targets

```diff
@@ -0,0 +1,126 @@
+            </AdditionalDependencies>
+
+            <!-- x86-SGX-Simulation-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Simulation-Debug' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">
+                $(SgxSimX86Dbg)\oeenclave.lib;$(SgxSimX86Dbg)\oesocket_enc.lib;$(SgxSimX86Dbg)\oestdio_enc.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x64-SGX-Simulation-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Simulation-Debug' And '$(Platform)' == 'x64'">
+                $(SgxSimX64Dbg)\oeenclave.lib;$(SgxSimX64Dbg)\oesocket_enc.lib;$(SgxSimX64Dbg)\oestdio_enc.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x86-ARMTZ-Simulation-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'ARMTZ-Simulation-Debug' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">
+                $(TzSimX86Dbg)\oeenclave.lib;$(TzSimX86Dbg)\oesocket_enc.lib;$(TzSimX86Dbg)\oestdio_enc.lib;$(TzSimX86Dbg)\oeenclave_opteesim.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+        </Link>
+    </ItemDefinitionGroup>
+
+    <!-- Linking Rules for Host Applications -->
+    <ItemDefinitionGroup Condition="'$(OEType)' == 'Host'">
+        <Link>
+            <!-- x86-SGX-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Debug' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">
+                $(SgxHwX86Dbg)\oehost.lib;$(SgxHwX86Dbg)\oesocket_host.lib;$(SgxHwX86Dbg)\oestdio_host.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x64-SGX-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Debug' And '$(Platform)' == 'x64'">
+                $(SgxHwX64Dbg)\oehost.lib;$(SgxHwX64Dbg)\oesocket_host.lib;$(SgxHwX64Dbg)\oestdio_host.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x86-SGX-Simulation-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Simulation-Debug' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">
+                $(SgxSimX86Dbg)\oehost.lib;$(SgxSimX86Dbg)\oesocket_host.lib;$(SgxSimX86Dbg)\oestdio_host.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x64-SGX-Simulation-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Simulation-Debug' And '$(Platform)' == 'x64'">
+                $(SgxSimX64Dbg)\oehost.lib;$(SgxSimX64Dbg)\oesocket_host.lib;$(SgxSimX64Dbg)\oestdio_host.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- arm-ARMTZ-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'ARMTZ-Debug' And '$(Platform)' == 'ARM'">
```

Comment:
I don't think the "ARMTZ-" prefix is useful in configuration names, it's redundant with the Platform value.

Reply:
Specifically, we should use the configuration names that are typical in visual studio projects.  So just "Debug"

Reply:
See my comment on distinguishing between ARM TrustZone and OP-TEE above.

As for just "Debug", we can't do that since we need to be able to distinguish between architecture (x86, x64), TEE (SGX, TZ), configuration (Debug, Release) and potentially TEE runtime (OP-TEE, something else in the future). CMake generates a complete build system for each combination, so just "Debug" would be ambiguous.

Reply:
We need to make sure this is easy to use by people who don't use cmake, so should default to what VS defaults to